source/settings/settings-reference.txt

```diff
@@ -86,6 +86,17 @@ You can configure the following settings on the |compass| interface:
 
        To learn more, see :ref:`compass-enable-dev-tools`.
 
+   * - Install |compass-short| as URL Protocol Handler
+     - General
+     - Register |compass-short| as a handler for mongodb:// and mongodb+srv:// 
+       URLs. 
+
+       If :guilabel:`Install Compass as URL Protocol Handler` is enabled, 
+       you can open |compass-short| by navigating to a mongodb:// or 
+       mongodb+srv:// URL in your browser.
+
+        *Available on macOS and Windows.*
```

Comment:
Just wondering whether this might be misread by users/if there’s a better phrasing – the _preference_ is that’s available on macOS and Windows, the _feature_ is also available on Linux.

Reply:
Ah I misunderstood that myself. Updating!